src/router/orders/validation.ts

```diff
@@ -14,17 +14,31 @@ export const ordersPost: OrdersPost = async function (req, res, next) {
     if (Object.keys(order).length < 1) {
       throw new ValidationError('Order is empty');
     }
-    if (!isObject(order.items)) {
+    if (!Array.isArray(order.items)) {
       throw new ValidationError('Invalid order');
     }
+    if (!isArrayOfObjects(order.items)) {
+      throw new ValidationError('Invalid order items');
+    }
 
-    const orderValues = Object.values(order.items);
-    const isOrderValuesValid = orderValues.every(
-      (value) => typeof value === 'number'
-    );
+    if (order.items.length < 1) {
+      throw new ValidationError('Invalid order');
+    }
 
-    if (orderValues.length < 1 || !isOrderValuesValid) {
-      throw new ValidationError('Invalid order item');
+    for (const item of order.items) {
+      const isOrderItemsValid = isObject(item.additives);
+      const orderAdditiveValues = Object.values(item.additives);
+      const isOrderAdditiveValuesValid = orderAdditiveValues.every(
+        (value) => typeof value === 'number'
+      );
+
+      if (
+        !isOrderItemsValid ||
```

Comment:
it's better to separate this check. In case when `isOrderItemsValid = false` the code brakes  when tries to take `Object.values` form not an object.